protocol/storage/miner_test.go

```diff
@@ -114,10 +114,26 @@ func TestReceiveStorageProposal(t *testing.T) {
 		assert := assert.New(t)
 		require := require.New(t)
 
-		_, miner, proposal := newMinerTestSetup()
+		porcelainAPI, miner, proposal := newMinerTestSetup()
 		proposal.Payment.Vouchers = []*paymentbroker.PaymentVoucher{}
 
-		res, err := miner.receiveStorageProposal(context.Background(), proposal)
+		// since we are modifying the deal after we create it, we must resign it
```

Comment:
Blocking: putting this re-signing inline in the test obscures its purpose (especially since it's duplicated below). Instead, make the number of vouchers a parameter of `newMinerTestSetup` or the methods that it calls here.